tsconfig.json

```diff
@@ -0,0 +1,14 @@
+{
+	"extends": "@sindresorhus/tsconfig",
+	"compilerOptions": {
+		"outDir": "dist",
+		"target": "es2018",
+		"lib": [
+			"es2018"
+		]
+	},
+	"include": [
+		"source",
+		"test"
```

Comment:
Because of https://github.com/sindresorhus/conf/pull/104/files#r383791352, `test` can be dropped here (don't forget to update `main` again in package.json).